README.md

```diff
@@ -16,73 +16,44 @@
 
-or
+We assume some knowledge of the following topics, though, initially, it may be sufficient to glance through a brief introduction using Google.
 
-    pip install -r requirements.txt
+1. **`Anaconda`** is a distribution of the Python language containing many useful Python packages. To offset its many advantages, it's quite large (about 2Gb). But that is not too much space for a modern home computer.
+2. **`Conda`**is an app, found in Anaconda, giving convenient management of large software packages.
+3. When working on a computer, one works in a certain *environment*, in which certain commands and procedures make sense. When working on large **Python** projects, such as ours, it is often convenient to work in a *virtual environment*. These virtual environments are controlled by `conda`. 
+4. Github: the site where this repository is found.
+5. git, a language giving control of different versions of the same program. A modern novelist might find it useful for keeping different versions of a novel, but usually it's used for software development.
 
-2.Clone the tiatoolbox
-
-    git clone https://github.com/TIA-Lab/tiatoolbox.git
-
-3.Please try
-
-    python -m tiatoolbox -h
+Getting Started
+===============
 
-pip
----
+1. Make a directory (also known as a *folder*) into which you will download this repository. You can call it anything you like. We will call it **tb**.
 
-    pip install tiatoolbox
+2. Open a terminal window, while in **tb**, for inputting commands.
 
-conda
------
+3. Clone (download a copy) of  the official repository for `tiatoolbox`using 
+   `conda clone https://github.com/TIA-Lab/tiatoolbox.git` 
+   
+4. Change into the new subdirectory `tb/tiatoolbox`.
 
-    conda env create -n tiatoolbox python=3.{6,7,8}
-    conda activate tiatoolbox
-    pip install tiatoolbox
+5. Create the virtual environment `tiatoolbox` with
+   `conda env create -f requirements.conda.yml`
+   (The name `tiatoolbox` for the environment is hidden inside the requirements file.)
+   
+6. `conda activate tiatoolbox`
 
-tiatoolbox --help
-=================
+   You now have several different objects called *tiatoolbox*, among them the directory in which your terminal is operating, and the environment *tiatoolbox* , created by conda.
 
-    usage: tiatoolbox [-h] [--version] [--verbose VERBOSE]
-                     {read-bounds,
-                      save-tiles,
-                      slide_info,
-                      slide-thumbnail,
-                      stainnorm,
-                      }
-                     ...
+7. Check success with `conda list`, which lists all the Python packages in the `tiatoolbox` virtual environment and also the version of Python that will be used in that environment.
 
-    positional arguments:
-      {slide_info}
+The above steps have installed the package `tiatoolbox`and created a function, also called `tiatoolbox`, whose usage is explained when you give the command
```

Comment:
Might want to rephrase this or add an extra step as the toolbox has been cloned but not actually installed (this would be done with setup.py or pip).

Reply:
@John-P Are you sure you are right about whether tiatoolbox is installed as a program? I'm can't be 100% sure, but I believe that I carried out ONLY the steps I have mentioned in the rewritten README.md. I have just done "conda activate tiatoolbox" in a bash window, and then
tiatoolbox --version
tiatoolbox --help
Both gave sensible answers, so it IS installed.
The README.md commands need to be tested for a new user, perhaps with nothing installed, not even Python. Is this something that one could set up an automatic test for, like for a Python program? If so, could you send me a link explaining how to set up a test that would be run whenever the README page is changed?
I just checked: /Users/dbae/opt/anaconda3/envs/tiatoolbox/bin has 337 entries. However,
ls | grep -i wsi  gives null response. Is that relevant? Is that what you mean by your comment?

Reply:
I think @John-P 's comment is good. I would like to include a command or commands for installing the Python package `tiatoolbox` but don't know how to do it.

Reply:
I think Shan's review now tells me explicitly what to do.